src/Symfony/Bundle/FrameworkBundle/Tests/Translation/TranslatorTest.php

```diff
@@ -180,7 +194,10 @@ public function testInvalidOptions()
         (new Translator($container, new MessageFormatter(), 'en', [], ['foo' => 'bar']));
     }
 
-    /** @dataProvider getDebugModeAndCacheDirCombinations */
+    /**
+     * @dataProvider getDebugModeAndCacheDirCombinations
+     * @group legacy
+     */
     public function testResourceFilesOptionLoadsBeforeOtherAddedResources($debug, $enableCache)
```

Comment:
With these changes, the following option is now unsupported

```php
'resource_files' => ['some_locale' => ['messages.some_locale.loader']]
```

so a work-around will be needed for this scenario (Maybe adding a new option set loaders per locale?)